exp/templates/exp/dashboard.html

```diff
@@ -16,7 +16,7 @@
       <h1>{% trans "Dashboard" %}</h1>
     </div>
     <div class="panel-body">
-      <p>You've successfully signed into Experimenter! </p>
+      <p>You've successfully signed into Researcher! </p>
```

Comment:
keep as Experimenter (referring to app) although to be fair I think this is unused